webpack.config.node.js

```diff
@@ -73,7 +73,33 @@ module.exports = {
 	},
 	plugins: [
 		// Require source-map-support at the top, so we get source maps for the bundle
-		new webpack.BannerPlugin( 'require( "source-map-support" ).install();', { raw: true, entryOnly: false } )
+		new webpack.BannerPlugin( 'require( "source-map-support" ).install();', { raw: true, entryOnly: false } ),
+		new webpack.NormalModuleReplacementPlugin( /^my-sites\/controller$/, 'lodash/utility/noop' ),
```

Comment:
We need a better solution for this. Perhaps we could include isomorphic and non-isomorphic middlewares/controllers using a package.json approach. We'd eventually get rid of any non-isomorphic stuff there.